tests/src/python/test_authmanager_pki_postgres.py

```diff
@@ -37,7 +37,11 @@
     QgsWKBTypes,
 )
 
-from PyQt4.QtNetwork import QSslCertificate
+
+try:
+    from PyQt4.QtNetwork import QSslCertificate
+except ImportError:
+    from PyQt5.QtNetwork import QSslCertificate
```

Comment:
I didn't even try, for some reason I've stopped to use `from PyQt import ...` since I've had mixed results when running tests locally. Is that working on master and the major backport branches?
But let's see what Travis thinks about it :smile: